json-path/src/test/java/com/jayway/jsonpath/internal/function/SequentialPathFunctionTest.java

```diff
@@ -50,4 +55,21 @@ public void testIndexOfText() throws Exception {
         verifyFunction(conf, "$.text.index(-1)", TEXT_SERIES, "f");
         verifyFunction(conf, "$.text.index(1)", TEXT_SERIES, "b");
     }
+
+    @Test
+    public void testDistinctOfText() throws Exception {
+        verifyFunction(conf, "$.text.distinct()", TEXT_SERIES, Arrays.asList("a", "b", "c", "d", "e", "f"));
+    }
+
+    @Test
+    public void testDistinctOfObjects() throws Exception {
+        final Object expectedValue = new ObjectMapper().readValue("[{\"a\":\"a-val\"}, {\"b\":\"b-val\"}]", Object.class);
```

Comment:
shouldn't `conf.mappingProvider()` be used instead of explicit creation of `ObjectMapper`?

Reply:
Thanks. Updated